adhocteam/Head-Start-TTADP
# Add additional buttons at bottom of activity report

**Description of change**
 * Split up saving of the report to the API and navigation
 * No longer save to API when navigating pages (still save after 2 minutes)
 * Add "save draft" button that saves to API
 * Continue renamed "Save & Continue"
 * Can save draft from the submitter draft review page

**How to test**
1) Pull down
2) Use the new buttons in an activity report (make sure to have at least one recipient). Data only hits the API when "Save & Continue", "Save draft" are pressed and when the 2 minute auto-save triggers.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/268

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
